sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/PropertiesUtils.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.aad.webapp;
+
+import org.springframework.boot.test.context.FilteredClassLoader;
+import org.springframework.boot.test.context.runner.WebApplicationContextRunner;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.RequestEntity;
+import org.springframework.security.oauth2.server.resource.BearerTokenAuthenticationToken;
+import org.springframework.util.MultiValueMap;
+
+import java.util.Optional;
+
+public class PropertiesUtils {
+
+    public static final WebApplicationContextRunner CONTEXT_RUNNER = new WebApplicationContextRunner()
```

Comment:
It should be a function, return a `new WebApplicationContextRunner()` on each call.